smodels/tools/simplifiedLikelihoods.py

```diff
@@ -963,11 +982,13 @@ def chi2(self, marginalize=False):
         nsig = self.model.convert(nsig)
 
         # Compute the likelhood for the null hypothesis (signal hypothesis) H0:
-        llhd = self.likelihood(nsig, marginalize=marginalize, nll=True, mu = 1. )
+        llhd = self.likelihood_beta(1., marginalize=marginalize, nll=True)
 
         # Compute the maximum likelihood H1, which sits at nsig = nobs - nb
         # (keeping the same % error on signal):
         if len(nsig) == 1:
+            # TODO this nsig initiation seems wrong and changing maxllhd to likelihood_beta
+            # fails ./testStatistics.py : zero division error in L115
             nsig = self.model.observed - self.model.backgrounds
             maxllhd = self.likelihood(nsig, marginalize=marginalize, nll=True, mu = 1. )
```

Comment:
There is a problem with this `nsig` initiation. `likelihood_beta` fails the `testStatistics.py`.